src/frontend/js/types/locale.ts

```diff
@@ -0,0 +1,25 @@
+// languages code come from django
+export type languageCode = 'en' | 'es' | 'fr' | 'fr-ca';
+
+// mapping between the language code and the local with the region
+export enum localesMapping {
+  fr = 'fr_FR',
+  'fr-ca' = 'fr_CA',
+  es = 'es_ES',
+  en = 'en_US',
+}
+
+// mapping between the language code and the locale data from react-intl
+export enum languageCodeMapping {
+  fr = 'fr',
+  'fr-ca' = 'fr',
+  es = 'es',
+  en = 'en',
```

Comment:
I'm not confortable with this. I don't understand why we can't use the same locales in the frontend and backend.
Let's discuss IRL.

Reply:
The main issue come from crowdin configuration. I didn't found how to have the same code when files are downloaded, if I can fix this issue we will don't have mapping issue, I already start removing all these file, we can deal without all of this.

Reply:
I found a solution but we have to modify a bit the backend application. If we switch the code (`fr`) by a locale (`fr-fr`) I can reorganize translated files and everything is now dynamic. Let's talk about his tomorrow.